cli/src/display.rs

```diff
@@ -424,7 +434,11 @@ fn create_table(
         let values = row.values();
         let mut v = vec![];
         for value in values {
-            v.push(value.to_string());
+            if replace_newline {
+                v.push(value.to_string().replace('\n', "\\n"));
```

Comment:
should `\r` also be replaced?

Reply:
and what about control characters like `\b` or `\v`

Reply:
I think just ‘\n' is ok, this is to control the display.